library/maven/rules.bzl

```diff
@@ -18,10 +18,13 @@ def maven(artifacts_org, artifacts_repo={}):
             "https://repo1.maven.org/maven2",
             "https://repo.vaticle.com/repository/maven",
             "https://repo.vaticle.com/repository/maven-snapshot",
+            "https://dl.google.com/dl/android/maven2",
+            "https://maven.pkg.jetbrains.space/public/p/compose/dev",
         ],
         strict_visibility = True,
         version_conflict_policy = "pinned",
-        fetch_sources = True
+        fetch_sources = True,
+        fail_on_missing_checksum = fail_on_missing_checksum
```

Comment:
By default, `maven` looks for the presence of a `SHA` or `MD5` hash, but many Compose dependencies don't have one - for example:
```
checksum errors: SHA-1: not found: https://maven.pkg.jetbrains.space/public/p/compose/dev/org/jetbrains/compose/ui/ui-text-desktop/1.0.0-alpha3/ui-text-desktop-1.0.0-alpha3.pom.sha1, MD5: not found: https://maven.pkg.jetbrains.space/public/p/compose/dev/org/jetbrains/compose/ui/ui-text-desktop/1.0.0-alpha3/ui-text-desktop-1.0.0-alpha3.pom.md5
```
In order to fetch these dependencies we have to disable `fail_on_missing_checksum` for the Studio repo.